libs/components/src/lib/text-area/text-area.scss

```
@@ -29,7 +29,7 @@ $outline-width: 2px;
 	@include connotation.connotation (text-area);
 
 	display: inline-grid;
-	width: inherit;
+	width: 100%;
```

```suggestion
	inline-size: 100%;
```
if we are changing - lets ditch the width :)

---

Sure, updated 👍